bucket/netbeans.json

```diff
@@ -57,9 +57,7 @@
             ]
         }
     },
-    "checkver": {
-        "regex": "Apache NetBeans ([\\d.]+)"
-    },
+    "checkver": "Apache\\sNetBeans\\s([\\d]+)",
     "autoupdate": {
         "url": "https://www.apache.org/dist/netbeans/netbeans/$version/netbeans-$version-bin.zip",
```

Comment:
```suggestion
        "url": "https://archive.apache.org/dist/netbeans/netbeans/$version/netbeans-$version-bin.zip",

```

Reply:
Please change the autoupdate url and use the checkver.ps1 tool locally to update the version to 17

Reply:
Okay, I've just done it